### PR TITLE
Add disk operation

### PR DESCRIPTION
**Pull request by imcvampire**
Close https://github.com/Kamatera/terraform-provider-kamatera/issues/12

### CODE REVIEW

**Review by OriHoch — CHANGES_REQUESTED** [review]
code looks good, but please add unit tests as we discussed, testing the code only without making actual requests to the api. The unit tests should include possible edge-cases and show which operations are being done based on which terraform configs.

File: kamatera/disk_test.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+			o: []interface{}{123.0},
+			n: []interface{}{124.0, 456.0},
+			expected: diskOperation{add: []float64{456}, update: map[int]float64{0: 124}},
+		},
+		{
+			name: "update and remove",
+			o: []interface{}{123.0, 456.0},
+			n: []interface{}{124.0},
+			expected: diskOperation{remove: []int{1}, update: map[int]float64{0: 124}},
+		},
```
Comment (OriHoch) [inline kamatera/disk_test.go:46]: please add some more test cases for edge-cases like:
* both disks having the same size
* 3 / 4 disks
* error cases

I'm especially worried about cases where we might accidentially delete customer data.. maybe it's worth to also check how it is handled in other terraform drivers

**Review by OriHoch — CHANGES_REQUESTED** [review]
excellent! the tests are great and exactly what we need

see comment

File: kamatera/disk_test.go (hunk 1 above)
Comment (imcvampire) [reply kamatera/disk_test.go:46]: Good catch! I will do it now.

Comment (imcvampire) [reply kamatera/disk_test.go:48]: Some new cases are added. Please check them.

**Review by OriHoch — APPROVED** [review]
:+1:
